TestFoundation/TestNSFileManager.swift

```diff
@@ -137,6 +138,26 @@ class TestNSFileManger : XCTestCase {
         }
     }
     
+    func test_setFileAttributes() {
+        let path = "/tmp/testfile"
```

Comment:
I stole this from the `get` test that was already there.

Can you recommend another design, and I can fix both of them?